timesketch/views/auth.py

```diff
@@ -265,22 +265,30 @@ def validate_api_token():
     validated_email = token_json.get('email')
 
     # Check if the authenticating user is part of the allowed domains.
-    domain_whitelist = current_app.config.get('GOOGLE_OIDC_HOSTED_DOMAIN')
-    if domain_whitelist:
+    hosted_domains = current_app.config.get('GOOGLE_OIDC_HOSTED_DOMAIN')
+    if hosted_domains:
         _, _, domain = validated_email.partition('@')
-        if domain.lower() != domain_whitelist.lower():
+        if domain.lower() != hosted_domains.lower():
             return abort(
                 HTTP_STATUS_CODE_UNAUTHORIZED,
                 'Domain {0:s} is not allowed to authenticate against this '
                 'instance.'.format(domain))
 
-    user_whitelist = current_app.config.get('GOOGLE_OIDC_USER_WHITELIST')
-    # Check if the authenticating user is on the whitelist.
-    if user_whitelist:
-        if validated_email not in user_whitelist:
+    allowed_users = current_app.config.get('GOOGLE_OIDC_ALLOWED_USERS')
+    # TODO: Remove that after a 6 months, this following check is to ensure
+    # compatibility of config file
+    if len(user_allowlist) == 0:
+        print("Warning, GOOGLE_OIDC_USER_WHITELIST has been deprecated. "
+                  "Please update timesketch.conf.")
```

Comment:
single vs double quotes

Reply:
also probably better to use the logger instead of print

Reply:
done, used logging, not sure if correct usage in auth.py, and needed to import logging in the analyzer, but that seems reasonable.